packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/apple/appidpinokta.md

```diff
@@ -0,0 +1,10 @@
+In the **General Settings** section:
+
+* **Name** &mdash; Enter the name that you would expect to see on a button, such as **Sign in with Apple**.
+* **Client ID** &mdash; Paste the Services ID that you obtained from Apple when you configured the Identity Provider in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>. The Services ID identifies the particular instance of your app and is used as the OAuth `client_id`.
+* **Client Secret Signing Key** &mdash; Click **Browse files** and locate the key file (.p8 format) that you downloaded from Apple when you created the private key for client authentcation in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>.
+* **Key ID** &mdash; Enter the Key ID that you obtained from Apple when you created the private key for client authentication in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>. The Key ID represents the private key generated by Apple.
+* **Team ID** &mdash; Enter your Apple Team ID. This value must be the 10 character (alphanumeric) Team ID from your Apple developer account. This ID can be found in the upper-right corner of your Apple developer dashboard just below your account name.
```

Comment:
We do not need to mention it is 10 character, Apple might change it in the future.